src/conn/conn_compact.c

```diff
@@ -18,6 +21,63 @@ __compact_server_run_chk(WT_SESSION_IMPL *session)
     return (FLD_ISSET(S2C(session)->server_flags, WT_CONN_SERVER_COMPACT));
 }
 
+/*
+ * __find_next_uri --
+ *     Given a URI, find the next one in the metadata file that is eligible for compaction.
+ */
+static int
+__find_next_uri(WT_SESSION_IMPL *session, const char *uri, const char **next_uri)
```

Comment:
`next_uri` --> `next_urip`

Output parameters should end with `p`.

Reply:
Sure, see fd1cbf88479516d7e54dbcc4fee28e00b7d588f2 and 0b262a95fc4e8ad5656e1c4b170a39229019e245.